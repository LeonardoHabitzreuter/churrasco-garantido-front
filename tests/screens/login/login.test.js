import React from 'react'
import { shallow } from 'enzyme'
import Login from '../../../src/screens/login'

describe('Login component', () => {
  test('should renders properly', () => {
    const wrapper = shallow(<Login />)
    expect(wrapper).toMatchSnapshot()
  })
})
