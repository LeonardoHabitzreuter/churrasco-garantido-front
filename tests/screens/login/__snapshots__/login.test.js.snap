// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component should renders properly 1`] = `
<Route
  render={[Function]}
/>
`;
